resources/clj-kondo.exports/manifold/manifold/config.edn

```diff
@@ -0,0 +1,8 @@
+{
+ ;; aliasing loop to let is enought to get
+ ;; great linting.
+
+
+ :lint-as {manifold.deferred/let-flow clojure.core/let
+           manifold.deferred/loop clojure.core/let}
```

Comment:
Shouldn't manifold's loop alias core loop?

Reply:
The issue with making manifold's loop alias to core loop is that kondo expects core loop to have an actual `recur` and emits warnings on the lack of it, while manifold uses `d/recur`. It is actually better to alias loop to let, since all we want is to let kondo know that the variable is being defined. From kondo's [docs](https://github.com/clj-kondo/clj-kondo/blob/master/doc/hooks.md#transformation):

> It is not important that the code is rewritten exactly according to the macroexpansion. What counts is that the transformation rewrites into code that clj-kondo can understand.

So as far as the linter is concerned, `d/loop` can be linted as `let`. One alternative to that is writing a kondo hook that translates  `d/recur` into `recur`, but that doesn't work very well either since `d/recur` is often used inside lambdas, where ordinary `recur` wouldn't work well for core loop (possibly causing more lint errors).

Reply:
Ahh, interesting. That makes sense.